### ISSUE TITLE
Translate: the accuracy of api is different than google translate app

### ISSUE THREAD

**Issue by smilence86** [googleapis/google-cloud-java#6562]
Thanks for stopping by to let us know something could be better!

**PLEASE READ**: If you have a support contract with Google, please create an issue in the [support console](https://cloud.google.com/support/) instead of filing on GitHub. This will ensure a timely response.

Please run down the following list and make sure you've tried the usual "quick fixes":

  - Search the issues already opened: https://github.com/googleapis/google-cloud-java/issues
  - Check for answers on StackOverflow: http://stackoverflow.com/questions/tagged/google-cloud-platform

If you are still having issues, please be sure to include as much information as possible:

#### Environment details

1. Specify the API at the beginning of the title (for example, "BigQuery: ...")
   General, Core, and Other are also allowed as types  
    translate(sourceText, srcLang, tgtLang, "'nmt")
2. OS type and version: ubuntu
3. Java version: openjdk version "1.8.0_222"
4. google-cloud-java version(s): google-cloud-translate 1.93.0

#### Steps to reproduce

  1. ?
  2. ? 

#### Code example

```java
// example
public String translate(String sourceText,
                            String sourceLang, String targetLang)
    {
        TranslateOption srcLang = TranslateOption.sourceLanguage(sourceLang);
        TranslateOption tgtLang = TranslateOption.targetLanguage(targetLang);

        Translation translation = translator.translate(
            sourceText, srcLang, tgtLang, model);

        return sourceText + "<newline>translator: " + translation.getTranslatedText();
    }
```

#### Stack trace
```
Any relevant stacktrace here.
```

#### External references such as API reference guides used	

- ?	

#### Any additional information below	
The same source words: "安卓手机很卡顿", API's translate result is:  
![image](https://user-images.githubusercontent.com/6102831/67460299-373b1d80-f66d-11e9-9893-41a496320007.png)
  
The google app's result is:  
![image](https://user-images.githubusercontent.com/6102831/67460439-913be300-f66d-11e9-821e-eede2264fb1f.png)
  
  
Change to other words: "苹果手机很卡顿", API's translate result is:  
![image](https://user-images.githubusercontent.com/6102831/67460497-b7618300-f66d-11e9-9661-05a75739ea03.png)

The google app's result is:  
![image](https://user-images.githubusercontent.com/6102831/67460532-cc3e1680-f66d-11e9-8a65-be68c618af86.png)

Same service by google, Why its different, It seems that app's accuracy is higher than api.

Thanks!

**Comment by codyoss** [comment]
Hey @smilence86 there are many features features in the translate api. I am not seeing in the documentation anywhere that the app you are referring to uses this api exactly. There are also AutoML models, Custom models, and pre-trained models that all fall under the translate product.

**Comment by smilence86** [comment]
@codyoss In my project, I used google-cloud-translate:
```
<dependency>
      <groupId>com.google.cloud</groupId>
      <artifactId>google-cloud-translate</artifactId>
      <version>1.93.0</version>
    </dependency>
```
It's the newest version from https://mvnrepository.com/artifact/com.google.cloud/google-cloud-translate. And the code:
```
private static final TranslateOption model = TranslateOption.model("nmt");
public String translate(String sourceText,
                            String sourceLang, String targetLang)
    {
        TranslateOption srcLang = TranslateOption.sourceLanguage(sourceLang);
        TranslateOption tgtLang = TranslateOption.targetLanguage(targetLang);

        Translation translation = translator.translate(sourceText, srcLang, tgtLang, model);

        return sourceText + "<newline>translator: " + translation.getTranslatedText();
    }
```
If neural network model effect accuracy that translate chinese into english, I want to know which model is the best to use. The result will as same as app.
